src/styles/components/_video-player.scss

```diff
@@ -16,8 +16,8 @@ $mc-video-progress-height: 0.8rem !default;
 
   &__video {
     position: absolute;
-    left: 50%;
-    top: 50%;
+    left: 50% !important;
+    top: 50% !important;
```

Comment:
adding `.vjs-tech` to the player shifted the video element based on styles that come from Brightcove.  this specificity overrides those new styles